hydrus/data/migrations/README.md

```diff
@@ -0,0 +1,90 @@
+
+Alembic - Database Migration Tool
+===================
+This repository is a pre-configured extension of Alembic to facilitate hydrus DB Migrations. [Alembic](https://alembic.sqlalchemy.org/en/latest/) is a lightweight database migration tool for usage with the SQLAlchemy Database Toolkit for Python.
+
+-----------------
+First make sure you set up hydrus as the [instructions here](https://github.com/HTTP-APIs/hydrus/blob/master/README.md).
+
+## Basic demo migration flow 
+
+With the default hydrus clone run it with a sqlite database:
+
+    hydrus serve --dburl sqlite:///hydrus/data/database.db
+
+#### Point to your DB Url(only if not using sqlite):
```

Comment:
Even when using sqlite, the DB name is a variable, so this text needs to be changed.

Reply:
Yes, I guess that's a bit confusing. I'll make a change.